cmscontrib/updaters/update_from_1.5.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+-- https://github.com/cms-dev/cms/pull/1378
+ALTER TYPE public.feedback_level ADD VALUE 'oi_restricted';
```

Comment:
Do we want to use the `public.` schema prefix everywhere? It's the default and we don't use any other schemata.